configure-buildfarm/tasks/main.yml

```diff
@@ -101,17 +101,17 @@
      - config
 
 -
-  name: "Create podtemplate update Groovy script from template"
+  name: "Create Kubernetes plugin config update Groovy script from template"
   template:
-    src: podtemplate-config.j2
-    dest: "/tmp/podtemplate-config.groovy"
+    src: kubernetes-plugin-config.j2
```

Comment:
It is renamed. I am now thinking if there is another thing that I need to rename.